components/src/core/header/headerActionItems.tsx

```diff
@@ -67,16 +73,29 @@ export const HeaderActionItems: React.FC<ActionItemProps> = (props) => {
     if (items) {
         return (
             <View style={[defaultStyles.root, styles.root]}>
-                {items.slice(0, 3).map((actionItem, index) => (
-                    <TouchableOpacity
-                        key={`action_${index}`}
-                        testID={`header-action-item${index}`}
-                        onPress={actionItem.onPress}
-                        style={[defaultStyles.actionItem, styles.actionItem]}
-                    >
-                        <HeaderIcon IconClass={actionItem.icon} />
-                    </TouchableOpacity>
-                ))}
+                {items.slice(0, MAX_ITEMS).map((actionItem: any, index: number) => {
```

Comment:
does actionItem need to be 'any' here? Or can we call it HeaderIconType | HeaderAvatar and cast where needed?

Reply:
Good call...We shouldn't need it now. I missed that in the cleanup.